azure-pipelines.yml

```diff
@@ -64,6 +47,20 @@ jobs:
       filename: script\download-spark-distros.cmd
       arguments: $(Build.BinariesDirectory)
 
+  - script: build.cmd -pack
```

Comment:
Use the -ci flag, it cleans up stuff before and after building, plus adds `RepoRoot\.dotnet` to the path.

Reply:
Done.